sdk/generic-sdk-core/generic-core-http-okhttp/src/main/java/com/generic/core/http/okhttp/implementation/OkHttpResponse.java

```diff
@@ -68,6 +71,24 @@ static HttpHeaders fromOkHttpHeaders(Headers okHttpHeaders) {
         return httpHeaders;
     }
 
+    /**
+     * Gets the {@link BinaryData} that represents the body of the response.
+     *
+     * @return The {@link BinaryData} containing the response body.
+     */
+    @Override
+    public BinaryData getBody() {
+        if (body == null) {
+            if (super.getValue() == null) {
+                body = super.getBody();
+            } else {
+                body = super.getValue();
```

Comment:
Does this mean the body would be the raw response here ?
We should probably clarify in the docs.

Reply:
In the case of this class, both the `value` and the `body` properties are of type `BinaryData`, which means we can treat them interchangeably for most purposes. This method mirrors what the parent class does, which is lazily assigning a value to `body` if the body has not been assigned already.

That said, your comment made me realize there's no need to call the `getBody()` method in the superclass, as we can simply return an empty body (a `BinaryData` created from an empty byte array) in that particular case.